charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-ui.yaml

```diff
@@ -480,7 +480,6 @@ spec:
           image: {{ printf "%s/%s/%s:%s" .Values.images.mysqlImage.registry .Values.images.mysqlImage.repo .Values.images.mysqlImage.imageName .Values.images.mysqlImage.tag }}
           imagePullPolicy: {{ .Values.images.pullPolicy }}
           args:
-            - "--ignore-db-dir=lost+found"
```

Comment:
@ayush-px Could you please share why this is not needed now, for my understanding.